### PR TITLE
Newspage

### PR DESCRIPTION
**Pull request by haattis**
Designed nyheter/[slug].tsx
Added new logo svgs
Various design fixes

### CODE REVIEW

File: components/FunkyArrow.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  font-size: 3em;
+  letter-spacing: -0.5em;
+  font-weight: bolder;
+  cursor: pointer;
+  transition: letter-spacing 1s ease;
+  -webkit-touch-callout: none; /* iOS Safari */
+  -webkit-user-select: none; /* Safari */
+  -khtml-user-select: none; /* Konqueror HTML */
+  -moz-user-select: none; /* Old versions of Firefox */
+  -ms-user-select: none; /* Internet Explorer/Edge */
```
Comment (oleast) [inline components/FunkyArrow.tsx]: I think we should be able to add these automatically?

File: components/Logo/LogoContainer.tsx [hunk 2]
```diff
@@ -5,19 +5,20 @@ const Wrapper = styled.div`
...
 const LogoImage = styled.img`
-  width: 100%;
   height: 100%;
 `;
 
 export const LogoContainer = () => {
   return (
     <Wrapper>
-      <LogoImage src="/Logo2.svg" alt="FIF logo" />
+      <a href="/">
```
Comment (oleast) [inline components/Logo/LogoContainer.tsx]: Should probably use `Link` from `next/link` or something link that?

File: public/Pil.svg [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        stroke: #accafc;
+        stroke-width: 5px;
+      }
+    </style>
+  </defs>
+  <g id="Pil" transform="translate(1.88)">
+    <line id="Line_4" data-name="Line 4" class="cls-1" y2="51.408" transform="translate(19.186 0)"/>
+    <path id="Path_6" data-name="Path 6" class="cls-1" d="M908.314,3379.132l19.286,22.006,19.293-22.006" transform="translate(-908.315 -3346.369)"/>
+  </g>
+</svg>
```
Comment (oleast) [inline public/Pil.svg]: Where is this used?

File: components/FunkyArrow.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+
+const FunkyArrow = styled.a`
```
Comment (oleast) [inline components/FunkyArrow.tsx]: Should also use the `Link` from the next package.
There is probably a guide out there for using it with styled-components

File: components/AboutMembers/index.tsx [hunk 5]
```diff
@@ -23,7 +23,9 @@ export const AboutMembers = () => {
       <Header color={colors.lightBlue}>Studentforeningene i FIF</Header>
       {MEMBERS.map(member => (
         <p key={member.link}>
-          <MemberLink href={member.link}>{member.name}</MemberLink>
+          <Link href={"http://online.ntnu.no"} passHref>
```
Comment (oleast) [inline components/AboutMembers/index.tsx]: ?

File: components/ArticleSummary.tsx [hunk 6]
```diff
@@ -25,14 +26,17 @@ const ArticleSummary: React.FC<ArticleSummaryInterface> = ({
   title,
   content
 }) => {
+  console.log(href);
```
Comment (oleast) [inline components/ArticleSummary.tsx]: Remove this

File: pages/_app.tsx [hunk 7]
```diff
@@ -15,6 +15,11 @@ const CustomApp = (appProps: Props): JSX.Element => {
       <Head>
         <title>Forente IT-Foreninger</title>
       </Head>
+      <link
```
Comment (oleast) [inline pages/_app.tsx]: I think we are supposed to use the _light_ version of this font, at least some places. Should probably include that as well?

File: pages/nyheter/[slug].tsx [hunk 8]
```diff
@@ -30,13 +47,17 @@ const BlogTemplate = ({ article }: Props) => {
...
         <Header>{article.data.title}</Header>
         <div>
           <ReactMarkdown source={article.content} />
         </div>
-        <a href="/nyheter">tilbake</a>
-      </article>
+      </Article>
+      <TopLeft>
+        <Link href="/nyheter">
+          <FunkyArrow color={colors.lightBlue}>{"<------"}</FunkyArrow>
```
Comment (oleast) [inline pages/nyheter/[slug].tsx]: Is it possible to just include the `<------` in the FunkyArrow component itself?

File: components/AboutMembers/index.tsx (hunk 5 above)
Comment (haattis) [reply components/AboutMembers/index.tsx]: Woops. Var en del av testingen

**Pull request merged**